react/features/prejoin/actions.web.ts

```diff
@@ -216,7 +216,7 @@ export function initPrejoin(tracks: Object[], errors: Object) {
  */
 export function joinConference(options?: Object, ignoreJoiningInProgress = false,
         jid?: string, password?: string) {
-    return async function(dispatch: IStore['dispatch'], getState: IStore['getState']) {
+    return function(dispatch: IStore['dispatch'], getState: IStore['getState']) {
```

Comment:
Doesn't this require for the flow to wait before resuming execution https://github.com/jitsi/jitsi-meet/pull/14907/files#diff-b057c3bfbdca09d0a54a610e23d6893becffec425f59299aeadf62187261ed6cR246?

Reply:
Hmmm. I don't see await in this function. The await that you pointed out is wrapped in a callback that is async (https://github.com/jitsi/jitsi-meet/pull/14907/files/99e21b0e2267f89bc905187092488c14e4af6e1e#diff-b057c3bfbdca09d0a54a610e23d6893becffec425f59299aeadf62187261ed6cR233). So I think all should be good. Isn't it?